dotnet-desktop-guide/net/wpf/migration/index.md

```diff
@@ -1,28 +1,25 @@
 ---
-title: Upgrade a WPF app to .NET 7
-description: Learn how to upgrade a .NET Framework (or previous .NET) Windows Presentation Foundation (WPF) application to .NET 7.
-ms.date: 06/01/2023
+title: Upgrade a WPF app to .NET 8
+description: Learn how to upgrade a .NET Framework (or previous .NET) Windows Presentation Foundation (WPF) application to .NET 8.
+ms.date: 02/07/2024
 ms.topic: how-to
 ---
 
-# How to upgrade a WPF desktop app to .NET 7
+# How to upgrade a WPF desktop app to .NET 8
 
-This article describes how to upgrade a Windows Presentation Foundation (WPF) desktop app to .NET 7. Even though WPF runs on .NET, a cross-platform technology, WPF is still a Windows-only framework. The following WPF-related project types can be upgraded with the .NET Upgrade Assistant:
+This article describes how to upgrade a Windows Presentation Foundation (WPF) desktop app to .NET 8. Even though WPF runs on .NET, a cross-platform technology, WPF is still a Windows-only framework. The following WPF-related project types can be upgraded with the .NET Upgrade Assistant:
 
 - WPF project
 - Control library
 - .NET library
 
 If you're upgrading from .NET Framework to .NET, consider reviewing the [Differences with WPF .NET](differences-from-net-framework.md) article and the [Porting from .NET Framework to .NET](/dotnet/core/porting/) guide.
 
-> [!WARNING]
-> Don't upgrade Visual Basic WPF projects. There seems to be a bug with the extension. This article will be updated when the bug is fixed.
```

Comment:
@adegeo This was removed for WinForms so I assumed it could be removed here too.